docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -51,10 +51,12 @@ To delete existing identity from filesystem realm:
 /subsystem=elytron/filesystem-realm=fsRealm:remove-identity(identity=alex)
 ----
 
-It's possible to update the ``key-store`` or ``key-store-alias`` to use for integrity checking using the ``update-key-pair`` command as shown below. This command will also update the signature in any existing identity files.
+It's possible to update the ``key-store`` or ``key-store-alias`` to use for integrity checking using the ``update-key-pair`` command as shown below. These commands will also update the signature in any existing identity files.
 [source, options="nowrap"]
 ----
-/subsystem=elytron/filesystem-realm=fsRealm:update-key-pair(key-store=newKeyStore, key-store-alias=newKeyStoreAlias)
+/subsystem=elytron/filesystem-realm=keystore:write-attribute(name=key-store, value=newKeystore)
```

Comment:
Just to be consistent with the name of the `filesystem-realm` used in the other examples, we should use `filesystem-realm=fsRealm` here instead of `filesystem-realm=keystore`.